packages/dashboard/src/components/DashboardFrame/index.tsx

```diff
@@ -173,7 +177,8 @@ export const PageFrame = memo((props: PageFrameProps) => {
                     </NavigationDrawer>
                 )}
                 <ShapeHelper>
-                    <ShapeContainer>
+                    <ShapeContainer
+                        className={props.noBackgroundFill ? undefined : classes.shapeContainerWithBackground}>
```

Comment:
You can use [classnames](https://www.npmjs.com/package/classnames)

Reply:
I think this is ok for now